cvat/apps/engine/models.py

```diff
@@ -331,6 +332,18 @@ def choices(self):
     def __str__(self):
         return self.value
 
+class ReviewStatus(str, Enum):
+    ACCEPTED = 'accepted'
+    REJECTED = 'rejected'
+    REVIEW_FURTHER = 'review_further'
```

Comment:
What is the initial state? Probably REVIEWER_FUTHER -> OPEN.

Reply:
Initial state of Review instance?

Reply:
In which state when you "open" a review? What is the lifecycle of review?

```
OPEN -> ACCEPTED
  |
 \ /
REJECTED
```
 
Can I change status from `ACCEPTED` to `REJECTED` without explanation immediately? The usual lifecycle of issues includes some intermediate states like `REOPENED`. When you open an issue, it is in `OPEN` state. Do we need them here?

Reply:
Review instance is considered like a static instance. Once created it cannot be changed.

At the same users can submit as many reviews as they want. For example the first review can have "rejected" status and task status goes from "review" to "annotations", after annotations were changed the task status goes to "review" again and getting review with "accepted" status, goes from "review" to "completed"